src/write.rs

```diff
@@ -0,0 +1,55 @@
+use std::fmt;
+use Event;
+
+#[derive(Debug)]
+pub struct Error;
+
+impl From<fmt::Error> for Error {
+  fn from(_: fmt::Error) -> Error {
+    Error
+  }
+}
+
+pub trait Write {
```

Comment:
Isn't it better to name this `HtmlWrite` or `EventWrite` to avoid confusion?

I'm already confused by `fmt::Write` and `io::Write` :sweat_smile: